Hanging daily release

## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Documentation issue or request
</code></pre>

## Description

Example in build [#527](https://ci.fabric8.io/job/syndesis-release-daily/527/), looks very similar to the memory problem we had previously with #5468.

Seems like the cause of the issue is same as in #5468 (from build [#529](https://ci.fabric8.io/job/syndesis-release-daily/529/)):

```
09:49:17 [ERROR] $ NODE_OPTIONS=--max-old-space-size=2048 craco build
09:49:17 [ERROR] Creating an optimized production build...
09:49:17 [ERROR] 
09:49:17 [ERROR] <--- Last few GCs --->
09:49:17 [ERROR] 
09:49:17 [ERROR] [13485:0x3342bf0]   346502 ms: Mark-sweep 1940.0 (2099.0) -> 1930.7 (2100.5) MB, 2163.9 / 0.0 ms  (average mu = 0.085, current mu = 0.025) allocation failure scavenge might not succeed
09:49:17 [ERROR] [13485:0x3342bf0]   348634 ms: Mark-sweep 1943.1 (2100.5) -> 1933.8 (2103.0) MB, 2103.9 / 0.0 ms  (average mu = 0.051, current mu = 0.013) allocation failure scavenge might not succeed
09:49:17 [ERROR] 
09:49:17 [ERROR] 
09:49:17 [ERROR] <--- JS stacktrace --->
09:49:17 [ERROR] 
09:49:17 [ERROR] ==== JS stack trace =========================================
09:49:17 [ERROR] 
09:49:17 [ERROR]     0: ExitFrame [pc: 0x1d397f15be1d]
09:49:17 [ERROR]     1: StubFrame [pc: 0x1d397f153ee5]
09:49:17 [ERROR] Security context: 0x18be4949e6e1 <JSObject>
09:49:17 [ERROR]     2: split [0x18be494906c1](this=0x3e3ea9ceb4e1 <String[305]: /mnt/hudson_workspace/workspace/syndesis-release-daily/app/ui-react/node_modules/react-scripts/node_modules/babel-loader/lib/index.js??ref--6-oneOf-2!/mnt/hudson_workspace/workspace/syndesis-release-daily/app/ui-react/packages/ui/node_modules/patternfly-reac...
09:49:17 [ERROR] 
09:49:17 [ERROR] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
09:49:17 [ERROR] 
09:49:17 [ERROR] lerna ERR! yarn run build stderr:
09:49:17 [ERROR] FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
09:49:17 [ERROR]  1: 0x8dc1c0 node::Abort() [/mnt/hudson_workspace/workspace/syndesis-release-daily/app/ui-react/node/node]
09:49:17 [ERROR]  2: 0x8dc20c  [/mnt/hudson_workspace/workspace/syndesis-release-daily/app/ui-react/node/node]
09:49:17 [ERROR]  3: 0xad60ae v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/mnt/hudson_workspace/workspace/syndesis-release-daily/app/ui-react/node/node]
09:49:17 [ERROR]  4: 0xad62e4 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/mnt/hudson_workspace/workspace/syndesis-release-daily/app/ui-react/node/node]
09:49:17 [ERROR]  5: 0xec3972  [/mnt/hudson_workspace/workspace/syndesis-release-daily/app/ui-react/node/node]
09:49:17 [ERROR]  6: 0xec3a78 v8::internal::Heap::CheckIneffectiveMarkCompact(unsigned long, double) [/mnt/hudson_workspace/workspace/syndesis-release-daily/app/ui-react/node/node]
09:49:17 [ERROR]  7: 0xecfb52 v8::internal::Heap::PerformGarbageCollection(v8::internal::GarbageCollector, v8::GCCallbackFlags) [/mnt/hudson_workspace/workspace/syndesis-release-daily/app/ui-react/node/node]
09:49:17 [ERROR]  8: 0xed0484 v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/mnt/hudson_workspace/workspace/syndesis-release-daily/app/ui-react/node/node]
09:49:17 [ERROR]  9: 0xed30f1 v8::internal::Heap::AllocateRawWithRetryOrFail(int, v8::internal::AllocationSpace, v8::internal::AllocationAlignment) [/mnt/hudson_workspace/workspace/syndesis-release-daily/app/ui-react/node/node]
09:49:17 [ERROR] 10: 0xe9c574 v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationSpace) [/mnt/hudson_workspace/workspace/syndesis-release-daily/app/ui-react/node/node]
09:49:17 [ERROR] 11: 0x113beae v8::internal::Runtime_AllocateInNewSpace(int, v8::internal::Object**, v8::internal::Isolate*) [/mnt/hudson_workspace/workspace/syndesis-release-daily/app/ui-react/node/node]
09:49:17 [ERROR] 12: 0x1d397f15be1d 
09:49:17 [ERROR] error Command failed with exit code 1.
09:49:17 [ERROR] 
09:49:17 [ERROR] lerna ERR! yarn run build exited 1 in '@syndesis/syndesis'
09:49:18 [ERROR] error Command failed with exit code 1.
09:49:18 [INFO] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Looks like we'll need to increase the memory again, what do you think about 2.5GiB max heap @syndesisio/ui?

--- a/app/ui-react/syndesis/package.json
+++ b/app/ui-react/syndesis/package.json
@@ -47,7 +47,7 @@
     "replay:dev-server": "BACKEND=http://localhost:8556 yarn proxy:setup; BACKEND=http://localhost:8556 PROXY_NO_WS=true yarn start",
     "replay": "npm-run-all -p replay:dev-server \"mock-server -- {1}\" --",
     "start": "BROWSER=false HOST=0.0.0.0 craco start",
-    "build": "NODE_OPTIONS=--max-old-space-size=2048 craco build",
+    "build": "NODE_OPTIONS=--max-old-space-size=3072 craco build",
     "test": "craco test",
     "e2e": "cypress open",
     "e2e:terminal": "cypress run",

